lib/benchee/benchmark/runner.ex

```diff
@@ -101,7 +105,7 @@ defmodule Benchee.Benchmark.Runner do
   end
 
   defp measure_runtimes(scenario, context, run_time, fast_warning)
-  defp measure_runtimes(scenario, _, 0, _), do: scenario
+  defp measure_runtimes(scenario, _, 0, _), do: []
```

Comment:
Found this bug in working on this. `measure_runtimes/4` now returns a list, not a Scenario struct. I figured that an empty list was enough, and the modifications I made here ended up catching this bug with existing tests, so I didn't think I needed to add a new one.